src/main/webapp/i18n/en/error.json

```
@@ -48,6 +48,7 @@
         "submissionPolicyActiveNull": "Submission policies must be activated or deactivated. Activation cannot be null.",
         "submissionPolicyUpdateFailedPolicyNotExist": "The submission policy could not be updated, because the programming exercise does not have a submission policy.",
         "oneActivePlagiarismCheck": "Please wait until the current plagiarism check in this course has finished until you start another one.",
+        "invalidPersonalAccessTokenLifetime": "Requested token lifetime exceeds the maximum lifetime for personal access tokens.",
```

```suggestion
        "invalidPersonalAccessTokenLifetime": "The requested token lifetime exceeds the maximum lifetime for personal access tokens.",
```
Consider adapting the text accordingly